Homework1Task1/Task1/ListInsertionSortExtension.cs

```diff
@@ -0,0 +1,33 @@
+namespace Task1;
+
+public static class ListInsertionSortExtension
+{
+    /// <summary>
+    ///     Sorts list in-place using insertion sort(O(n^2) complexity) using provided comparer.
+    /// </summary>
+    public static void InsertionSort<T>(this List<T> list, IComparer<T> comparer)
```

Comment:
```suggestion
    public static void InsertionSort<T>(this List<T> list) where T : IComparable<T>
```
Просто хочу убедиться, что ты знаешь о такой возможности указать, что типовой параметр реализует интерфейс.